### PR TITLE
Initial support for dart-mx8-mini

### PR DESCRIPTION
**Pull request by acostach**
TODO:
- bluetooth not working yet
- need to check HUP
- other changes that might be machine-specific, for other derived machines

### CODE REVIEW

File: imx8mm-var-dart.coffee [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        IMX8M_VAR_DART_POST_FLASH
+        instructions.BOARD_REPOWER
+]
+ 
+module.exports =
+        version: 1
+        slug: 'imx8mm-var-dart'
+        name: 'Variscite DART-MX8M Mini'
+        arch: 'aarch64'
+        state: 'released'
```
Comment (floion) [inline imx8mm-var-dart.coffee]: This would be `new` here

File: imx8mm-var-dart.coffee [hunk 2]
```diff
@@ -0,0 +1,54 @@
+deviceTypesCommon = require '@resin.io/device-types/common'
+{ networkOptions, commonImg, instructions } = deviceTypesCommon
+ 
+IMX8M_VAR_DART_FLASH = 'Set the SW7 BOOT SELECT switch to SDCARD. Insert SD CARD. Power up the <%= TYPE_NAME %>.'
+IMX8M_VAR_DART_POST_FLASH = 'Set the SW7 BOOT SELECT switch to eMMC.'
```
Comment (floion) [inline imx8mm-var-dart.coffee]: I think the pcb has either `ext` or `int` on this boot switch?

File: imx8mm-var-dart.coffee (hunk 1 above)
Comment (acostach) [reply imx8mm-var-dart.coffee]: Thanks Florin, solved

File: imx8mm-var-dart.coffee (hunk 2 above)
Comment (acostach) [reply imx8mm-var-dart.coffee]: Correct, solved

**Pull request merged**